service-onboarding-build-pack/Jenkinsfile

```diff
@@ -289,7 +290,9 @@ node  {
 		{
 			try{
 				withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: var_credentialsId, passwordVariable: 'PWD', usernameVariable: 'UNAME']]) {
-					createProjectInSCM(env.gitlab_private_token, env.gitlab_cas_name_space_id, repo_base, scm_repo_name, uname, pwd, cas_rest_repo)
+					loadScmModule(buildModuleUrl, uname, pwd)
+					def cas_repo_id = scmModule.getCasRepoId()
```

Comment:
The getCasRepoId() here is used to set the value of the cas_repo_id within the module itself; however, there are definitely better ways of handling this and this implementaton was just convenient at the time because I was migrating logic to the module. I'll have the module be more self dependent.